EapCommons/src/main/java/de/hpi/unicorn/attributeDependency/AttributeDependencyManager.java

```diff
@@ -140,6 +140,13 @@ public boolean removeAll(boolean force) {
 		return true;
 	}
 
+	public boolean removeDependency(AttributeDependency dependency) {
+		if(dependency.remove() == null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') 'if' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_Around_BP2016)